fortran/README.md

```diff
@@ -0,0 +1,55 @@
+# How to use Fortran interface
```

Comment:
Change this to developer's readme, i.e. notes on how to design the binding, what fortran version we use and what type of allocatable arrays we use.

We treat the example in the other readme

Reply:
Which do you mean developers of spglib fortran interface or developers who use spglib?

Reply:
Developers of spglib fortran interface

Reply:
I see. Probably you mean something like this 7cd1c5b.

Reply:
Well, change the title for starters:
```suggestion
# Fortran interface
```

Also remove `## Conceptual example`. Otherwise that looks ok